specs/packagemanager/PackageManagement.md

```diff
@@ -145,7 +148,23 @@ not ready can include:
 Is\*Ready() methods are a quick test to determine if more (costly) work is needed before the target
 can be used.
 
-## 3.3. Ensure\*Ready()
+## 3.3. Is\*ReadyOrNewerAvailable()
```

Comment:
This renumbering is causing a large ripple in the change. Future, just add a new number to the end of the list.